client/src/leap/soledad/client/sync.py

```diff
@@ -39,6 +39,7 @@ class SoledadSynchronizer(Synchronizer):
 
     Also modified to allow for interrupting the synchronization process.
     """
+    received_docs = []
```

Comment:
it doesn't need it, but I find quite convenient to just simply add a class-attribute by default, to prevent attribute erros,  and then let it be wiped out in the sync method. 
otherwise, we have to create a constructor and keep it in sync with the inherited class, which I think it can be more error prone.